css/base.scss

```diff
@@ -206,10 +206,34 @@ input[type=search] {
         }
       }
 
+      .methods {
+        @include display-flex();
+        @include flex-direction(row);
+        flex-wrap: wrap;
+        margin: 0 10px;
+
+        > p {
```

Comment:
It seems unnecessary to include the `<` in `< p {`, as there seems to be only one `<p>` tag under the "methods" class.